test/BaseTabs.test.ts

```diff
@@ -0,0 +1,42 @@
+import { test, expect } from 'vitest';
+import { render } from '@testing-library/vue';
```

Comment:
Why `testing-library`?

I would prefer if you use `@vue/test-utils` directly, because it is a quite good enough official Vue toolkit.